From 88c1e83e02a59f4811f06757daced6c821fa54d9 Mon Sep 17 00:00:00 2001
From: progier389 <progier@redhat.com>
Date: Mon, 11 Apr 2022 16:15:07 +0200
Subject: [PATCH] Issue 5254 - dscreate create-template regression due to
 5a3bdc336 (#5255)

dscreate create-template regression due to 829ea4113..5a3bdc336
  (default value for template_file parameter was unwillingly changed to 'None' (instead of None))

Issue: 5254   https://github.com/389ds/389-ds-base/issues/5254

Reviewed by:

(cherry picked from commit 45af34013f8bdd34f939d36b16776413e13c0a51)
---
 src/lib389/cli/dscreate | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/lib389/cli/dscreate b/src/lib389/cli/dscreate
index 0b41166cc..aa3878ff9 100755
--- a/src/lib389/cli/dscreate
+++ b/src/lib389/cli/dscreate
@@ -51,7 +51,7 @@ interactive_parser.set_defaults(func=cli_instance.instance_create_interactive)
 template_parser = subparsers.add_parser('create-template', help="Display an example inf answer file, or provide a file name to write it to disk.")
 template_parser.add_argument('--advanced', action='store_true', default=False,
     help="Add advanced options to the template - changing the advanced options may make your instance install fail")
-template_parser.add_argument('template_file', nargs="?", default='None', help="Write example template to this file")
+template_parser.add_argument('template_file', nargs="?", default=None, help="Write example template to this file")
 template_parser.set_defaults(func=cli_instance.instance_example)
 
 subtree_parser = subparsers.add_parser('ds-root', help="Prepare a root directory in which non root user can create, run and administer instances.")
-- 
2.37.3

